ShowBuildMenu.bat

```diff
@@ -15,7 +15,7 @@ echo.
 echo --- TESTING ---
 echo B. (Step 1) Set up a new test configuration for a particular database.
 echo C. (Step 2) Activate a test configuration.
-echo D. (Step 3) Run tests using active configuration.
+echo D. (Step 3) Run tests using active configuration (Build in Visual Studio).
```

Comment:
Precisely, I would use "built" for meaning this. It should have already been built in Visual Studio.